src/i18n/en.toml

```diff
@@ -95,9 +97,14 @@ titleParticipation = "With whom will you participate in the challenge?"
 hintPariticipation = "If you plan to compete alone, choose a group from which someone could possibly join your team later."
 
 [form.company]
+buttonAddAddress = "Add new"
+buttonAddBranch = "Add branch"
+buttonAddCompany = "Add company"
+hintAddress = "We will deliver a starter pack for your team to this address."
+labelAddress = "Company or branch address"
```

Comment:
Same as my comment above.

Reply:
Updated translation terms to "subdivision".